bin/avro2kafka

```diff
@@ -10,8 +10,8 @@ option_parser = OptionParser.new do |opts|
   opts.banner = "Version #{Avro2Kafka::VERSION} of Avro2Kafka\n" \
     "Usage: #{File.basename(__FILE__)} [options] [file]"
 
-  opts.on('-b', '--broker BROKER', 'The Kafka broker, eg. localhost:9092. This value is required.') do |broker|
-    options[:broker] = broker
+  opts.on('-b', '--broker-list BROKER', 'The Kafka broker list. Format: host:port. This value is required.') do |broker_list|
+    options[:broker_list] = broker_list
```

Comment:
Parsing and validation of the argument `broker_list` should be done here and only an array of brokers passed as `options`.